src/Gir/Marshal/SymbolTable.Builtin.cs

```diff
@@ -12,11 +12,12 @@ void RegisterBuiltIn (bool nativeWin64)
 			AddType (new StringMarshal ("gfilename", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToFilenamePtr({0})", "GLib.Marshaller.FilenamePtrToStringGFree({0})"));
 			AddType (new StringMarshal ("gchar", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
 			AddType (new StringMarshal ("char", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
+			AddType (new StringMarshal ("utf8", "string", "", "IntPtr"));//, "GLib.Marshaller.StringToPtrGStrdup({0})", "GLib.Marshaller.PtrToStringGFree({0})"));
```

Comment:
I just stubbed this out. I'm not sure what else we want here if anything?

Reply:
We can probaby remove more primitives as per https://wiki.gnome.org/Projects/GObjectIntrospection/Annotations/#Default_Basic_Types